azure-media/src/main/java/com/microsoft/windowsazure/services/media/authentication/AzureAdClientUsernamePassword.java

```diff
@@ -26,8 +26,8 @@ public String getPassword() {
 
     /**
      * Initializes a new instance of the AzureAdClientSymmetricKey class.
-     * @param clientId The client ID.
-     * @param clientKey The client key.
+     * @param username The client username
+     * @param password The client password
```

Comment:
Same issue here. Not a password, but a Key for an AD Application

Reply:
We also refer to this as Client ID and Client Key in most of our Media Services docs

Reply:
The point I'm making is that the argument into the method is called `password` - but the param mentioned in the javadoc is `clientKey` - they should be the same thing so that it matches up in the docs.

Reply:
docs should be right with clientKey and clientId

Reply:
Ok, I will invert the change, changing the method arguments to be clientId and clientKey